pycode/memilio-epidata/memilio/epidata/getVaccinationData.py

```diff
@@ -740,8 +735,16 @@ def get_vaccination_data(read_data=dd.defaultDict['read_data'],
         moving_average_sanit = 0
         impute_sanit = 'zeros'
 
+    # define new column names
+    column_names_dict = {
+        1: dd.EngEng['vaccPartial'],
+        2: dd.EngEng['vaccComplete'],
+        3: dd.EngEng['vaccRefresh'],
+        11: dd.EngEng['vaccNotComplete'],
+        'x': dd.EngEng['vaccRefresh']}
```

Comment:
The initialization for key x is confusing when you use the same argument as for key 3.
Also i think key 11 is confusing as well :D

Reply:
key `11` is provided from the RKI, see [here](https://github.com/robert-koch-institut/COVID-19-Impfungen_in_Deutschland#covid-19-impfdaten-auf-ebene-der-landkreise). 
Key `'x'` describes which column name (with the addition `_2`, `_3`...) should be used for numbers which are not in the `column_names_dict`. Do you have a suggestion to make this easier to read?